### PR TITLE
pycp: update to 7.3.0

### PR DESCRIPTION
**Pull request by ndowens**
python-cli-ui is a new package that is needed for pycp runtime

### CODE REVIEW

File: srcpkgs/pycp/template [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
 revision=1
 noarch=yes
 build_style=python3-module
-hostmakedepends="python3 help2man python3-setuptools"
-makedepends="python3-setuptools"
-depends="${makedepends}"
-short_desc="A cp and mv with a progressbar"
+pycompile_module="pycp"
+hostmakedepends="help2man python3-setuptools"
+depends="python python-cli-ui"
```
Comment (Hoshpak) [inline srcpkgs/pycp/template]: I think it should be python3 since the build_style is python3-module.

File: srcpkgs/python-cli-ui/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python-cli-ui'
+pkgname=python-cli-ui
+version=0.9.1
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="cli_ui"
+hostmakedepends="python3-setuptools"
+depends="python"
```
Comment (Hoshpak) [inline srcpkgs/python-cli-ui/template]: same here.

File: srcpkgs/python-cli-ui/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python-cli-ui'
+pkgname=python-cli-ui
```
Comment (Hoshpak) [inline srcpkgs/python-cli-ui/template]: If this module is for python3, the package name should reflect that.

**Comment by ndowens** [comment]
Done

**Comment by Hoshpak** [comment]
unless github shows me outdated data, the `pkgname=` line and the template comment still need to be changed.

**Comment by ndowens** [comment]
> unless github shows me outdated data, the `pkgname=` line and the template comment still need to be changed.

Woops forgot to change that part after got mv  will have it fixed real soon

**Comment by ndowens** [comment]
Believe to be fixed now

**Comment by Hoshpak** [comment]
It now needs to set `wrksrc` explicitely.

**Comment by ndowens** [comment]
Done

**Pull request merged**